content/00.front-matter.md

```diff
@@ -49,3 +49,12 @@ on {{manubot.date}}.
   {%- endif %}
   </small>
 {% endfor %}
+
+{% if manubot.authors|map(attribute='corresponding')|select|first %}
+  To whom correspondence should be addressed:
+  {% for author in manubot.authors %}
+    {% if author.corresponding is defined and author.corresponding is not none %}
+      {{author.initials}}: {{author.email}}
+    {% endif %}
+  {% endfor %}
```

Comment:
IIUC the indentation here is causing the corresponding author text to be treated as a code block:

> ![image](https://user-images.githubusercontent.com/1117703/178037522-9b56eee4-27e7-4278-b5e5-182d99d58dcd.png)

If we want a code block, I think the triple backtick syntax is more explicit. But a code block is probably the wrong formatting.

It might be worth looking through some of the existing corresponding author formats manuscripts have used (noted at https://github.com/manubot/rootstock/issues/469#issuecomment-1179199764) for inspiration on the best format for here.

Reply:
I agree that we probably want some other formatting besides a code block.